src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -373,11 +373,17 @@ public void verifyMonocolorAd(String slotName) {
 
   public AdsBaseObject waitForPageLoaded() {
     jsActions.waitForJavaScriptTruthy("document.readyState === 'complete'");
+    return this;
+  }
 
-    String waitForGPTJS = "typeof window.googletag === 'object'";
-    jsActions.waitForJavaScriptTruthy(waitForGPTJS);
+  public AdsBaseObject waitForPageLoaded(boolean withGPT) {
```

Comment:
Maybe rename this method to something like waitForPageLoadedWithGPT?
because for now `waitForPageLoaded()` almost equals `waitForPageLoaded(false)`